docs-v2/pages/workflows/settings.mdx

```diff
@@ -112,6 +112,14 @@ By default, workflows run with {process.env.MEMORY_LIMIT} of memory. If you're p
 
 Increasing your workflow's memory gives you a proportional increase in CPU, so increasing your workflow's memory can reduce its overall runtime and make it more performant.
 
+<Callout type="info">
+**Reducing Workflow Latency**
```

Comment:
I would use the same terminology you do in the troubleshooting guide: "How can my workflow run faster?"